default/scripting/focs/_effects.py

```diff
@@ -1,8 +1,19 @@
 # Placeholder for module.
+from typing import cast
+
 from focs._effects import (
+    LocalCandidate,
     StatisticIf,
 )
 
 
+def InGame():
+    """Returns a condition which matches objects on the map
+
+    :returns: a focs condition matching objects with positive ID
+    """
+    return cast(int, LocalCandidate.ID) > 0
```

Comment:
are you sure it should be `> 0` not `>= 0` ?

Reply:
fix in #5320